content/en/blog/2020/docker-rate-limit/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Handling Docker Hub rate limiting
+description: How to ensure your clusters are not impacted by Docker Hub rate limiting.
+publishdate: 2020-12-07
+attribution: John Howard (Google)
+keywords: [docker]
+target_release: 1.8
+---
+
+Since November 20th, 2020, Docker Hub has introduced [rate limits](https://www.docker.com/increase-rate-limits) on image pulls.
+
+Because Istio uses [Docker Hub](https://hub.docker.com/u/istio) as the default registry, usage on a large cluster may lead
+to pods failing to startup due to exceeding rate limits. This can be especially problematic for Istio, as there is typically
+the Istio sidecar image alongside most pods in the cluster.
+
+## Mitigations
+
+Istio allows you to specify a custom docker registry which you can use to make container images be fetched from your private registry. This can be configured by passing `--set hub=<some-custom-registry>` at installation time.
```

Comment:
This seems to imply a particular installation methodology. Is this instruction correct for an operator-based install, for example?

Reply:
We don't really have a consistent way to explain how to set an installation option. I have https://github.com/istio/istio.io/issues/4945 opened, but its quite old, to have a common abstraction in the docs for this. From the --set command a user using the operator could derive the IstioOperator yaml, although I am not sure all users know how to do this; --set is basically the same yaml API but flattened